sapi/cgi/cgi_main.c

```diff
@@ -2210,7 +2210,7 @@ consult the installation file that came with this distribution, or visit \n\
 		} else {
 			parent = 0;
 		}
-#endif /* WIN32 */
+#endif
```

Comment:
Replace the comment with `PHP_WIN32` instead maybe?

Reply:
Sure, coming up on ~Friday. I'm not at the computer ATM and I'm changing machines etc... 😄

Reply:
Added.